cdc/kv/region_worker.go

```diff
@@ -617,3 +785,35 @@ func (w *regionWorker) evictAllRegions(ctx context.Context) error {
 	}
 	return err
 }
+
+// InitWorkerPool initializs workerpool once, the workerpool must be initialized
+// before any kv event is received.
+func InitWorkerPool() {
+	if !enableKVClientV2 {
+		return
+	}
+	workerPoolOnce.Do(func() {
+		cfg := config.GetGlobalServerConfig().KVClient
+		var size int
+		if cfg.WorkerPoolSize > 0 {
+			size = cfg.WorkerPoolSize
+		} else {
+			size = runtime.NumCPU() * 2
+		}
```

Comment:
Is there any test that covers these lines?

Reply:
added unit test